src/app/dialogs/choose-template/choose-template.component.ts

```diff
@@ -0,0 +1,51 @@
+import { Component, Inject, OnInit } from '@angular/core';
+import { MAT_DIALOG_DATA, MatDialogRef } from '@angular/material/dialog';
+import { ApiLink } from 'src/app/services/api-data-types';
+import { PluginRegistryBaseService } from 'src/app/services/registry.service';
+import { TemplateApiObject } from 'src/app/services/templates.service';
+
+@Component({
+    selector: 'qhana-choose-template',
+    templateUrl: './choose-template.component.html',
+    styleUrls: ['./choose-template.component.sass']
+})
+export class ChooseTemplateComponent implements OnInit {
```

Comment:
Dialogs should end with `Dialog` (not `Component`). (please rename everything, including the file names...)
The purpose of this is to more easily identify Dialogs and other UI Components as such (also Services, etc.)